README.md

```diff
@@ -725,6 +726,30 @@ As an example this code has been tested with Keycloak, with the following variab
 
 See [GitLab documentation](https://docs.gitlab.com/ee/integration/oauth2_generic.html#sign-into-gitlab-with-almost-any-oauth2-provider) and [Omniauth-oauth2-generic documentation](https://gitlab.com/satorix/omniauth-oauth2-generic) for more details.
 
+##### OpenID Connect
+
+To enable OpenID Connect provider, you must register your application with your provider. You also need to confirm OpenID Connect provider app's ID and secret, the client options and the user's response structure.
+
+To use OIDC set at least `OAUTH_OIDC_ISSUER` and `OAUTH_OIDC_CLIENT_ID`.
+
+| GitLab setting | environment variable |
+|----------------|----------------------|
+| `label` | `OAUTH_OIDC_LABEL` |
+| `icon` | `OAUTH_OIDC_ICON` |
+| `scope`| `OAUTH_OIDC_SCOPE` |
+| `response_type` | `OAUTH_OIDC_RESPONSE_TYPE` |
+| `issuer` | `OAUTH_OIDC_ISSUER` |
+| `discovery` | `OAUTH_OIDC_DISCOVERY` |
+| `client_auth_method` | `OAUTH_OIDC_CLIENT_AUTH_METHOD` |
+| `uid_field` | `OAUTH_OIDC_UID_FIELD` |
+| `send_scope_to_token_endpoint` | `OAUTH_OIDC_SEND_SCOPE_TO_TOKEN_EP` |
+| `pkce` | `OAUTH_OIDC_PKCE` |
+| `client_options.identifier` | `OAUTH_OIDC_CLIENT_ID` |
+| `client_options.secret` | `OAUTH_OIDC_CLIENT_SECRET` |
+| `client_options.redirect_uri` | `OAUTH_OIDC_REDIRECT_URI` |
```

Comment:
Could you be so kind and add the default values defined in `assets/runtime/functions`? I guess, this will be helpful.

Reply:
I updated the README.md.